canister_ids.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't think you should add canister_ids.json to the repo, because it really depends on what type of network you're deploying to (mainnet, testnet), therefore the IDs can be different.

can you maybe workaround it in a way that it'll generate a needed canister_ids.json on the fly? and maybe to add it to `.gitignore`?